frontend/apps/reader/modules/readerfont.lua

```diff
@@ -887,6 +896,17 @@ a { color: black; }
 <h1>%s</h1>
 ]], _("Available fonts test document"), _("AVAILABLE FONTS")))
     local face_list = cre.getFontFaces()
+    if next(newly_added_fonts) then
+    -- Sort alphabetically, with new fonts first (as done in sortFaceList())
+        local move_idx = 1
```

Comment:
This comment reads odd: it should be indented too, similarly to the code in the `if` branch.